src/connection.cpp

```diff
@@ -417,6 +531,8 @@ namespace oxen::quic
         pkt_tx_timer_updater pkt_updater{*this, ts};
         size_t stream_packets = 0;
 
+        std::atomic<bool> prefer_big_first{true};
```

Comment:
Since only `flush_packets` touches this (and flush_packets will only ever be called inside the main thread) this ought to be safe as a plain `bool`